dds_web/utils.py

```diff
@@ -92,7 +92,7 @@ def invoice_units():
     """Get invoicing specification from Safespring, calculate and save GBHours and cost for each
     facility and project."""
 
-    app.logger.debug("Calculating invoicing info...")
+    current_app.logger.debug("Calculating invoicing info...")
```

Comment:
Change the import to `flask` and use `flask.current_app` like in the other files.

Reply:
Done! It was the only file where current_app was already imported, so I just used that one. Changed to `flask.current_app` in all the places now.